docs/r4ml-examples.md

```diff
@@ -5,13 +5,16 @@
    Congratulations, you just installed R4ML, now you are ready to do scalable
    data analysis and machine learning.
 
-   1) We have the comprehensive examples at [examples folder](../R4ML/inst/examples)
+   - We have comprehensive examples at [examples folder](../R4ML/inst/examples)
 
-   2) Our unit-test also provide great examples at [unittests](../R4ML/tests/testthat)
+   - Our unit-test also provides great examples at [unittests](../R4ML/tests/testthat)
+   
+#  - We have some vignettes for at [vignettes](../R4ML/vignettes)
```

Comment:
I md `#` means header. I think you have to use `<!--- stuff to comment out -->`

https://github.com/nilmeier/r4ml/blob/delete_vignettes/docs/r4ml-examples.md